infoscreen/templatetags/infoscreen_tags.py

```diff
@@ -1,16 +1,18 @@
 # imports
 from django import template
 from django.conf import settings
+
 # End: imports -----------------------------------------------------------------
 
 register = template.Library()
 
 # https://docs.djangoproject.com/en/3.0/howto/custom-template-tags/
 
-@register.inclusion_tag('infoscreen/components/screen.html')
+
+@register.inclusion_tag("infoscreen/components/screen.html")
 def screen_component(screen, *args, **kwargs):
-    default_classes = ''
+    default_classes = ""
     return {
-        'screen': screen,
-        'classes': kwargs.get('classes', default_classes),
-    }
\ No newline at end of file
+        "screen": screen,
+        "classes": kwargs.get("classes", default_classes),
+    }
```

Comment:
Jeg ser at det ikke er lagt inn ennå, men vi kommer til å gå for single quotes i linteren.

Reply:
En linter endrer ingenting automatisk tror jeg, kanskje det er en formatter? Har vi satt opp yapf i dette prosjektet?

Reply:
Yapf er satt opp ja, hadde Black med double quotes som standard i vscode, tror det var den skyldige

Reply:
skjønner skjønner